### PR TITLE
[snap/frontend]: init project

### PR DESCRIPTION
**Pull request by AnthonyLaw**
## What was the issue?
- New snap frontend project

## What's the fix?
- init project
- added jest unit test config
- added eslint
- Jenkins
- Docker

### CODE REVIEW

File: snap/frontend/README.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+- [Overview](#overview)
+- [Repository layout](#repository-layout)
+- Instructions
+    - [Installation](#installation)
+    - [Test](#test)
+    - [lint](#lint)
+
+## overview
+
+Snap fronted is a Symbol wallet application build on NextJS framework. It need connect to snap backend and interact with the Symbol blockchain.
```
Comment (Wayonb) [inline snap/frontend/README.md:12]: Snap fronted is a Symbol wallet application built on the NextJS framework. It must connect to the Snap backend to interact with the Symbol blockchain.

**Review by Wayonb — APPROVED** [review]
👍🏾

File: snap/frontend/.eslintrc [hunk 2]
```diff
@@ -0,0 +1,12 @@
+---
+extends:
+- eslint-config-next
+- plugin:import/react
+- ../../linters/javascript/default.eslintrc
```
[the remark this answers is not in the document]
Comment (AnthonyLaw) [reply snap/frontend/.eslintrc:5]: Should this plugin be added [here](https://github.com/symbol/symbol/blob/c9224646876ea899a93436ef3fe091632d710999/linters/javascript/default.eslintrc) or just for specify project?

File: snap/frontend/.eslintignore [hunk 3]
```diff
@@ -0,0 +1 @@
+build/
```
[the remark this answers is not in the document]
Comment (AnthonyLaw) [reply snap/frontend/.eslintignore:1]: I was having issues when running the build.
but it seems like it is not needed, as I removed it, it won't have an issue anymore.

**Review by Wayonb — APPROVED** [review]
👍🏾

**Review by Wayonb — APPROVED** [review]
👍🏾

**Pull request merged**